CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Minor: Allow resub messages to show in `/mentions` tab (#3148)
 - Minor: Added a setting to hide similar messages by any user. (#2716)
 - Minor: Duplicate spaces now count towards the display message length. (#3002)
+- Minor: Commands now use backup slots. (#3168)
```

Comment:
I think this fits better for something that simple users are gonna see
```suggestion
- Minor: Commands are now backed up as well. (#3168)
```
or maybe this, I'm not @felanbird

```suggestion
- Minor: Commands will have extra backups files as well. (#3168)
```